radixdlt-core/radixdlt/src/main/java/org/radix/GenerateUniverses.java

```diff
@@ -389,13 +390,13 @@ private static void outputUniverse(
 			byte[] universeBytes = serialization.toDson(u, Output.WIRE);
 			RadixAddress universeAddress = new RadixAddress((byte) u.getMagic(), k.getPublicKey());
 			RRI tokenRri = RRI.of(universeAddress, TokenDefinitionUtils.getNativeTokenShortCode());
-			if (!helmUniverseOutput.getOutputHelmValues() && !awsSecretsUniverseOutput.getEnableAwsSecrets()) {
+			if (!helmUniverseOutput.getOutputHelmValues() && !awsSecretsOutputOptions.getEnableAwsSecrets()) {
```

Comment:
This condition is repeated twice, so it might have sense to move it into dedicated  method with appropriate name.

Reply:
Done